_source/_posts/2019-07-10-java-microprofile-jwt-auth.md

```diff
@@ -265,18 +249,7 @@ Click **Create**.
 
 Because the example app is an API, it expects the request to come with a valid token and will not directly provide a way for the user to log in. There is no login redirect flow. How, then, do you get a token? OpenID Connect Debugger to the rescue! 
```

Comment:
```suggestion
Because the example app is an API, it expects the request to come with a valid token and will not directly provide a way for the user to log in. There is no login redirect flow. How, then, do you get a token? 
```